src/components/molecules/navLinks/NavLinks.tsx

```diff
@@ -1,22 +1,21 @@
-import React from 'react';
-import { ListItemButton, ListItemText } from '@mui/material';
-import { Link } from 'react-router-dom';
+import { ListItemButton, ListItemText } from '@mui/material'
+import { Link } from 'react-router-dom'
 
 interface NavLinksProps {
```

Comment:
🎨 Convention is to use `type` here. No real difference though.

Reply:
So 'type NavLinksProps' instead?

Reply:
Obviously `interface` works. But I prefer to use `type` for Component props and then `interface` for "db" structs

Reply:
This is a much more helpful explanation for me here on when to use type vs interface. Still new concept to me in TS